src/actions/auger/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Looker Data Sciences, Inc.
+
```

Comment:
@derekflint can you add a second line here after Looker's for Auger.AI? Example: `Copyright (c) 2019 Auger.AI`